k2/csrc/ragged_ops.h

```diff
@@ -216,31 +213,34 @@ std::vector<RaggedShape> GetPrefixes(RaggedShape &src,
                                      const std::vector<int32_t> &sizes);
 
 /*
-  Return a sub-part of a RaggedShape containing indexes 0 through n-1 of
-  its 1st axis.
-      @param [in] src  Source RaggedShape
-      @param [in] n    Number of (leading) indexes to keep; result will
-                       satisfy ans.Dim0() == n; Must have 0 <= n <= src.Dim0().
-      @return          Returns RaggedShape containing a prefix of `src`.
-                       It will share memory with `src`.
+  Append a single element to each sub-array of a ragged matrix (we consider
```

Comment:
We have duplicate declarations for `Prefix` and `GetPrefixes`, seems something got wrong when github did merging before.

Reply:
ah OK, please fix when convenient..